TodoApp.ConsoleApp/UI/TaskOperations.cs

```diff
@@ -90,15 +90,17 @@ static void TaskMaker()
         {
             string title = UserInput.ReadText("Enter a title: ", true);
             string message = UserInput.ReadText("Enter a description: ", true);
-            int deadLine = int.Parse(UserInput.ReadText("How many days you need to finish the task: ",true));
+#pragma warning disable CA1305 // Specify IFormatProvider
```

Comment:
Don't suppress the warning. Add `CultureInfo.CurrentCulture` as a second argument for `int.Parse()`.